libs/datamodel/core/src/transform/ast_to_dml/db/attributes.rs

```diff
@@ -179,25 +194,25 @@ fn visit_scalar_field_attributes<'ast>(
             id::field(ast_model, field_id, model_attributes, args, ctx)
         });
 
-         // @updatedAt
-         attributes.visit_optional_single("updatedAt", ctx, |args, ctx| {
-             if !matches!(scalar_field_data.r#type, ScalarFieldType::BuiltInScalar(tpe) if tpe.is_datetime()) {
-                 ctx.push_error(args.new_attribute_validation_error(
+        // @updatedAt
+        attributes.visit_optional_single("updatedAt", ctx, |args, ctx| {
+            if !matches!(scalar_field_data.r#type, ScalarFieldType::BuiltInScalar(tpe) if tpe.is_datetime()) {
+                ctx.push_error(args.new_attribute_validation_error(
                     "Fields that are marked with @updatedAt must be of type DateTime." ));
 
-             }
+            }
 
-             if ast_field.arity.is_list() {
-                 ctx.push_error(args.new_attribute_validation_error("Fields that are marked with @updatedAt cannot be lists."));
-             }
+            if ast_field.arity.is_list() {
+                ctx.push_error(args.new_attribute_validation_error("Fields that are marked with @updatedAt cannot be lists."));
+            }
 
-             scalar_field_data.is_updated_at = true;
-         });
+            scalar_field_data.is_updated_at = true;
+        });
 
-         // @default
-         attributes.visit_optional_single("default", ctx, |args, ctx| {
+        // @default
+        attributes.visit_optional_single("default", ctx, |args, ctx| {
             visit_field_default(args, scalar_field_data, model_id, field_id, ctx);
-         });
+        });
```

Comment:
I'm wondering where all these formatting problems come from in this file, it looks like rustfmt is giving up.